CONTRIBUTING.md

```diff
@@ -17,3 +21,48 @@ the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/pub
 All contributions to this project will be released under the CC0
 dedication. By submitting a pull request, you are agreeing to comply
 with this waiver of copyright interest.
+
+## Bugs
+
+### Known Issues
+
+We track issues on the GitHub [Issues page](https://github.com/ustaxcourt/ef-cms/issues). Before filing any new issues, please check to see if we've addressed the problem before or already have an existing issue related to it.
+
+### Reporting Issues
+
+From the [Issues page](https://github.com/ustaxcourt/ef-cms/issues) you can click the `New issue` button or go straight to the [list of templates](https://github.com/ustaxcourt/ef-cms/issues/new/choose).
+
+### Security-related Issues
+
+For security-sensitive issues, please do not file a public issue. Rather, please email [Dawson support](dawson.support@ustaxcourt.gov).
+
+## Proposing Changes
+
+We welcome pull requests, but we do still want those changes to be related to an [Issue](#reporting-issues). Please file one prior to submitting a pull request.
+
+## Pull Request Procedures
+
+Please follow these steps before submitting a pull request:
+
+1. Fork the repository and create a branch from `staging`.
+1. Run `npm install` in the repository root.
+   1. Do not delete `package-lock.json`.
+1. If reasonable for the changes being made, write tests.
+1. Run `npm run lint:fix` in the repository root.
+1. Ensure that all tests run with the following scripts pass:
+   1. `npm run test:api`
+   1. `npm run test:client`
+   1. `npm run test:shared`
+   1. `npm run test:pa11y`
+
+For non-trivial changes, PR's should be initiated into the `test` branch. Others can be initiated straight into the `staging` branch.
+
+### Active Pull Requests
+
+We will be civil, but we are opinionated about the architecture and coding standards in the project. Please do not be offended if the team requests changes or otherwise provides feedback. We may use [emoji's](https://github.com/erikthedeveloper/code-review-emoji-guide) in our comments.
```

Comment:
I wonder if we can incorporate our existing [code review documentation](https://github.com/ustaxcourt/ef-cms/blob/staging/docs/additional-resources/code-review.md). Some of it is outdated but a lot of it is good content, as it discusses the code review approach and what we look for which I appreciate in its explicitness.

It makes sense to me that you'd put a few sentences here under CONTRIBUTING, but maybe the language could be more like:

> As code reviewers, we provide feedback on pull requests so we can maintain the quality of our code, but do so with respect, kindness, and an open mind. We want code reviews to be a space for discussion and questions, while explaining our reasoning for suggestions.

Reply:
Probably should have explained my reasoning 😆  - I think it would be helpful to specify expectations for what we provide feedback for and that could even be integrated into how the code might be written in the first place. That's how some of the existing code review docs could be helpful in fleshing that out. The main part that I'm unsure about is the "please do not be offended" because to me it seems like we're putting the responsibility on them without clarifying the expectations enough around coding standards and architecture. I might be missing where the coding standards are further specified, though. Happy to talk about it irl if you want!

Reply:
@rosiehsmith, I'm coming back around to this. I'm definitely open to discussing changing this section. I had thought that at some point we would want to link to other documentation which is why I asked about whether we could get the github pages set up for the court repo to host docsify. Of course we can still link out if that is not going to happen.

Reply:
@jamesobrooks We [enabled docsify for the Court repo](https://ustaxcourt.github.io/ef-cms/). ✅

Reply:
@mmarcotte, thanks! I'll get a change pushed to refer to that documentation.